README.md

```diff
@@ -58,6 +58,10 @@ If you would like to configure your guests to be accessible from the host as sub
 
     config.landrush.tld = 'vm'
 
+Multiple TLD can also be provided as an array. On OS X, a resolver configuration will be created for each TLD.
```

Comment:
So what's the purpose here really? You need visibility of these TLD on the hosts, right? And as it stands now, this will only work for OS X out of the box. In fact even making the single TLD work out of the box works only for OS X atm.